lldb/source/API/SBInstructionList.cpp

```diff
@@ -165,10 +169,14 @@ bool SBInstructionList::GetDescription(Stream &sref) {
               addr, eSymbolContextEverything, sc);
         }
 
+        if (next_addr && addr != next_addr)
```

Comment:
I'm not the most experienced with optionals, doesn't the second `next_addr` need to be `*next_addr`?

Reply:
It doesn't *need* to be because `std::optional<T>` defines an operator== for comparison with plain `T`s (which treats an empty optional as different from all values of `T`), but yes, it would be better to compare the `T` directly since that's not the behavior I want (I basically want to treat the empty optional as *equal* to any address).